src/commands/mod.rs

```diff
@@ -171,8 +170,11 @@ enum ConfigSubcommand {
         /// An optional name of a specific config option to look up.
         #[arg(value_parser=NonEmptyStringValueParser::new())]
         name: Option<String>,
-        // TODO(#531): Support --show-origin using LayeredConfigs.
-        // TODO(#531): Support ConfigArgs (--user or --repo) and --all.
+        /// Whether to explicitly include built-in default values in the list.
+        #[arg(long)]
+        include_defaults: bool,
```

Comment:
BTW I'm thinking a little about how `--include-defaults` would combine with `--user`/`--repo`/etc... Simplest is probably to not let them be combined, have them all mutually exclusive with each other and with `--include-defaults`. I could imagine supporting a union like `config list --include-defaults --env --user`, but it feels a little awkward and I doubt anyone would ever need it.

Reply:
I think exclusive options are good. I agree it's unlikely user would want to see `--env --repo`.